src/Card/CardDeck.java

```diff
@@ -0,0 +1,133 @@
+package Card;
+
+import java.util.*;
+
+public class CardDeck {
+    
+    HashMap<String, String> cards;
+    // these are the numbers for the user that how many cards must have been asked
+    int easyCardNumber;
+    int hardCardNumber;
+    int harderCardNumber;
+    // variables for the easy, hard and harder maps
+    HashMap<String, String> easyCards;
+    HashMap<String, String> hardCards;
+    HashMap<String, String> harderCards;
+    
+    public CardDeck(){
```

Comment:
We could implement another constructor that waits for the numbers of cards by difficulty.